omnigibson/action_primitives/symbolic_semantic_action_primitives.py

```diff
@@ -523,7 +532,8 @@ def _cut(self, obj):
         # TODO: Do some more validation
         added_obj_attrs = []
         removed_objs = []
-        output = REGISTERED_RULES["SlicingRule"].transition({"sliceable": [obj]})
+        slicing_rule = REGISTERED_RULES["SlicingRule"]
+        output = REGISTERED_RULES["SlicingRule"].transition(slicing_rule, object_candidates={"sliceable": [obj]})
```

Comment:
Something is wrong in this line. Why do we refer to the same thing (REGISTERED_RULES["SlicingRule"]) in two different ways? You should be able to just remove the slicing_rule variable and that item in the transition function no? Is this a class method or an instance method?